coverage/index.html

```diff
@@ -14,7 +14,7 @@
       <img src="./assets/0.12.2/loading.gif" alt="loading"/>
```

Comment:
If you can remember, don't add your `coverage/` files, it's a pain to resolve conflicts and I don't need to pull down your `coverage` files.